city_scrapers/spiders/fortx_Fort_Worth_Isd_Coc.py

```diff
@@ -0,0 +1,112 @@
+from city_scrapers_core.constants import COMMITTEE
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+from dateutil.parser import parse
+
+
+class FortxFortWorthIsdCocSpider(CityScrapersSpider):
+    name = "fortx_Fort_Worth_Isd_Coc"
+    agency = "Fort Worth ISD Citizens' Oversight Committee"
+    timezone = "America/Chicago"
+    start_urls = [
+        "https://www.fwisd.org/departments/operations/capital-improvement-program/2021-citizens-oversight-committee-coc"  # noqa
+    ]
+
+    def parse(self, response):
+        """
+        `parse` should always `yield` Meeting items.
+
+        Change the `_parse_title`, `_parse_start`, etc methods to fit your scraping
+        needs.
+        """
+
+        location = {
+            "name": "Fort Worth ISD District Service Center",
+            "address": "7060 Camp Bowie Blvd, Fort Worth, TX 76116",
+        }
+
+        # go over the new events
+        for item in response.css(".fsDayContainer"):
+            meeting = Meeting(
+                title=self._parse_upcoming_title(item),
+                description="",
+                classification=COMMITTEE,
+                start=self._parse_upcoming_start(item),
+                end=self._parse_upcoming_end(item),
+                all_day=False,
+                time_notes="",
+                location=location,
+                links=[],
+                source=self._parse_source(response),
+            )
+
+            meeting["status"] = self._get_status(meeting)
+            meeting["id"] = self._get_id(meeting)
+
+            yield meeting
+
+        # go over the old events
+        # the first table has the old events
+        for item in response.css("table")[0].css("tr"):
+            meeting = Meeting(
+                title="2021 COC",
```

Comment:
I suggest setting the default meeting title to '2021 Citizens' Oversight Committee Meeting' to maintain consistency with the titles of future meetings.